envi/cli.py

```diff
@@ -783,6 +796,18 @@ def showmem():
         showmem()
         self.setEmptyMethod(showmem)
 
+    def validate_script(self, scriptpath):
+        try:
+            with open(scriptpath, 'rb') as f:
+                contents = f.read()
+
+            cobj = compile(contents, scriptpath, 'exec')
+            return True
+        except Exception, e:
+            pass
```

Comment:
please use os.path.isfile() to check the filepath for being a plain file and try/except wrap only the compile ( with the except catching a more specific exception type related to compile() failing for syntax).  

The construct "except Exception, e: pass" would silently return False if a future delta were to simply typo a variable name...